src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAcquireable.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Button, Serif } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { ArtworkSidebarAcquireable_artwork } from "__generated__/ArtworkSidebarAcquireable_artwork.graphql"
+export interface ArtworkSidebarAcquireableProps {
+  artwork: ArtworkSidebarAcquireable_artwork
+}
+
+export const ArtworkSidebarAcquireable: React.SFC<
+  ArtworkSidebarAcquireableProps
+> = props => {
+  const { artwork } = props
+  return (
+    <>
+      <Box pb={2}>
+        {artwork.is_sold ? (
```

Comment:
I believe the way things set up in the database/metaphysics is such that the Sold work will never be `acquirable`.

Don't think we need to check for any direct `is_something` fields here. I believe we should be able to use `sale_message` on artwork directly in all the cases (maybe skipping the auction case) and it should handle the data that marked in dark black font in Commercial component in design here: https://app.zeplin.io/project/5b113f413ede4bb10dbfdc42/screen/5bbfbf8ea55ed15fd3e9d957